common/src/main/java/org/opensearch/sdk/SdkClientSettings.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
I assume these settings will eventually move to core, right?

Otherwise we usually keep our settings in one place: https://github.com/opensearch-project/ml-commons/blob/0b9708b831f87ac6eb35b5c3039c61afe9425d26/plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java#L17

Reply:
Yes, these are intended to be moved to wherever this whole package will eventually move (an OpenSearch module?  Another plugin?  The SDK for Java extensions repo? But it will not remain in ML Commons long term and those settings names will probably change when we relocate it.